style.css

```diff
@@ -251,3 +251,158 @@ h3 {
   line-height: 24px;
   color: #000;
 }
+
+.grid {
+  background-color: #ebf0ee;
+  height: 352px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 20px;
+  flex: 1;
+  margin-left: auto;
+  margin-right: auto;
+  width: 19em;
+  border-radius: 10px;
+}
+
+.grid-image {
+  display: flex;
+  justify-content: center;
+}
+
+.tags1 {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  column-gap: 5px;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+  margin-top: -90px;
+  gap: 8px;
+  width: 271px;
+  height: 40px;
+  order: 1;
+}
+
+.tags2 {
+  display: flex;
+  flex-direction: row;
+  gap: 8px;
+  margin-top: 60px;
+}
+
+.txt1 {
+  margin-top: 70px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+}
+
+.tag1 {
+  padding: 5px;
+  border: 1px solid rgb(224, 224, 224);
+  margin-bottom: 5px;
+  border-radius: 8px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  color: rgb(6, 202, 137);
+}
+
+.contact_section {
+  padding: 150px 0 20px 0;
+  border-top-left-radius: 100px;
+  display: flex;
+  justify-content: center;
+  background: linear-gradient(#6070ff, rgba(48, 68, 252, 0.85)), url(./images/background2.jpg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: right 5px bottom 10%;
+}
```

Comment:
- [ ] Good job adding the background image in this section, I think there's some minor difference from the Figma design. Kindly check if the background image is getting applied correctly. 🥸️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/206780786-564998f9-087a-48d8-ad07-8c78ad7ea2a8.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/206780792-a497f741-f30b-4f41-87f1-929b7d1ea5c3.png" width="300">|